etc/config/c++.amazon.properties

```diff
@@ -383,6 +383,16 @@ compiler.raspbian9.exe=/opt/compiler-explorer/arm/raspbian9-6.3.0/bin/arm-raspbi
 compiler.raspbian9.name=Raspbian Stretch
 compiler.raspbian9.semver=6.3
 compiler.raspbian9.objdumper=/opt/compiler-explorer/arm/raspbian9-6.3.0/bin/arm-raspbian9-linux-gnueabihf-objdump
+compiler.arduinouno.exe=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-g++
+compiler.arduinouno.options=-std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -mmcu=atmega328p -DF_CPU=16000000L -DARDUINO=10809 -DARDUINO_AVR_UNO -DARDUINO_ARCH_AVR -I/opt/compiler-explorer/avr/arduino-1.8.9/hardware/arduino/avr/variants/standard -I/opt/compiler-explorer/avr/arduino-1.8.9/hardware/arduino/avr/cores/arduino
+compiler.arduinouno.name=Arduino Uno
+compiler.arduinouno.semver=5.4.0
+compiler.arduinouno.objdumper=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-objdump
+compiler.arduinouno.exe=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-g++
+compiler.arduinouno.options=-std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -mmcu=atmega2560 -DF_CPU=16000000L -DARDUINO=10809 -DARDUINO_AVR_MEGA2560 -DARDUINO_ARCH_AVR -I/opt/compiler-explorer/avr/arduino-1.8.9/hardware/arduino/avr/variants/mega -I/opt/compiler-explorer/avr/arduino-1.8.9/hardware/arduino/avr/cores/arduino
+compiler.arduinouno.name=Arduino Mega
```

Comment:
this uses the same ID as above: you'll need a different ID here, presumably you meant `arduionomega` -- I do this kind of thing all the time, it's the fault of the confusing configuration system....sorry. Can you also include the version too please.